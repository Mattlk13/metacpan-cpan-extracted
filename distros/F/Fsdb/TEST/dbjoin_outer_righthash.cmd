prog='dbjoin'
args='-m righthash -t outer --input TEST/dbjoin_outer.in-2 -n cid'
cmp='diff -c -b '
in=TEST/dbjoin_outer.in
