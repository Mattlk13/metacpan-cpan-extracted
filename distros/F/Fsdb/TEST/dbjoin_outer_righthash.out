#fsdb cid sid cname
10	1	pascal
11	2	numanal
12	1	os
12	2	os
13	3	-
14	-	theory
# cid 14: the class with no students
# cid 13: the unnamed class
#  | dbsort -n cid
#   | dbjoin -m righthash -t outer --input TEST/dbjoin_outer.in-2 -n cid
